### PR TITLE
MM-3651: Corrected several typos

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: Profiles - ZIB 2017/Valuesets/IHE-formatcode.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <id value="IHE.formatcode.vs"/>
       <text>
     <status value="additional" /><div xmlns="http://www.w3.org/1999/xhtml">
-	Important: this valueset has been copied from IHE. It is published by Nictiz through the IHE domain as to guarantee continuity and availability of this profile. The IHE FormatCode Value set is defined to be the set of <a href="http://wiki.ihe.net/index.php?title=IHE_Format_Codes">FormatCode(s) defined by IHE</a>.
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/Valuesets/IHE-formatcode.xml:5]: Is this the proper way of writing? Why not "ValueSet"?

File: Profiles - ZIB 2017/nl-core-careteam.xml [hunk 2]
```diff
@@ -14,8 +14,8 @@
       <use value="work" />
     </telecom>
   </contact>
-  <description value="A nl-core base CareTeam profile, that references to all relating Dutch Health and Care Information models or HCIM (Dutch: Zorginformatiebouwsteen or ZIB) profiles.  This profile is not based on a ZIB, beceause no ZIB exists for the CareTeam concept." />
-  <purpose value="The MedMij GGZ use case has a direct need voor exchanging careteam information with the patient." />
+  <description value="An nl-core base CareTeam profile, that references to all relating Dutch Health and Care Information models or HCIM (Dutch: Zorginformatiebouwsteen or ZIB) profiles. This profile is not based on a zib, beceause no zib exists for the CareTeam concept." />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/nl-core-careteam.xml]: "zib" should be "HCIM"

File: Profiles - ZIB 2017/zib-NutritionAdvice.xml [hunk 3]
```diff
@@ -209,7 +209,7 @@
       <path value="NutritionOrder.oralDiet.texture.modifier.text" />
       <short value="Consistency" />
       <definition value="A description of the consistency of the food." />
-      <comment value="Because of a granularity difference between the ZIB DieataryRecommendations and NutritionOrder, the ZIB concept Consistency is mapped to the elements .oralDiet.texture.modifier.text, .oralDiet.texture.foodType.text  and .oralDiet.fluidConsistencyType.text." />
+      <comment value="Because of a granularity difference between the zib NutritionAdvice and the NutritionOrder resource, the zib concept Consistency is mapped to the elements `.oralDiet.texture.modifier.text`, `.oralDiet.texture.foodType.text` and `.oralDiet.fluidConsistencyType.text`." />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-NutritionAdvice.xml]: "zib" should be "HCIM"

**Review by pieter-edelman-nictiz — CHANGES_REQUESTED** [review]
"zib" should be "HCIM" in the zib2017 repository.

File: Profiles - ZIB 2017/zib-AdministrationAgreement.xml [hunk 4]
```diff
@@ -277,7 +277,7 @@
       <path value="MedicationDispense.category.coding" />
       <sliceName value="administrationAgreementCode" />
       <short value="AdministrationAgreementCode" />
-      <definition value="Category snomed code used for indexing/retrieval of the ZIB AdministrationAgremeent. The snomed code 422037009 for this resource is stated in the Medication Process v09." />
+      <definition value="Category SNOMED code used for indexing/retrieval of the HCIM AdministrationAgreement. The SNOMED code 422037009 for this resource is stated in the Medication Process v09." />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-AdministrationAgreement.xml]: I think it's better to handle this change using a proper change request.

File: Profiles - ZIB 2017/zib-MedicationAdministration.xml [hunk 5]
```diff
@@ -261,7 +261,7 @@
       <path value="MedicationAdministration.category.coding" />
       <sliceName value="MedicationAdministrationCode" />
       <short value="MedicationAdministrationCode" />
-      <definition value="Category snomed code used for indexing/retrieval of the ZIB MedicationAdministration. The SNOMED CT code 422037009 for this resource is stated in the Medication Process v09." />
+      <definition value="Category SNOMED code used for indexing/retrieval of the HCIM MedicationAdministration. The SNOMED code 18629005 for this resource is stated in the Medication Process v09." />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-MedicationAdministration.xml]: I think it's better to use a proper change request for this

**Review by pieter-edelman-nictiz — CHANGES_REQUESTED** [review]
Overall, nice work! I think the two places where the SNOMED codes in the comment are aligned to the actual code can be better handled using a change request though, so that users are properly notified.

Also: I'm not sure of "value set" and "code system" are always the best choice; sometimes the sentence refers to the FHIR resource, other times to the concept in general (still better than "valueset" and "codesystem" though. so I wouldn't put in a lot of effort to correct this).

File: Profiles - ZIB 2017/zib-AdministrationAgreement.xml (hunk 4 above)
Comment (LuudSlagter) [reply Profiles - ZIB 2017/zib-AdministrationAgreement.xml]: See https://github.com/Nictiz/Nictiz-STU3-Zib2017/pull/307

File: Profiles - ZIB 2017/zib-MedicationAdministration.xml (hunk 5 above)
Comment (LuudSlagter) [reply Profiles - ZIB 2017/zib-MedicationAdministration.xml]: See https://github.com/Nictiz/Nictiz-STU3-Zib2017/pull/307

File: Profiles - ZIB 2017/conceptmap-AllergieStatusCodelijst-to-allergy-clinical-status.xml [hunk 6]
```diff
@@ -17,8 +17,8 @@
             <use value="work"/>
         </telecom>
     </contact>
-    <description value="Maps the HCIM AllergyIntolerance release 2017 terminology for AllergyStatus to the FHIR STU3 terminology"/>
-    <purpose value="To aid in mapping from HCIM AllergyIntolerance value set AllergieStatusCodelijst to FHIR STU3 AllergyIntolerance clinicalStatus and verificationStatus"/>
+    <description value="Maps the HCIM AllergyIntolerance release 2017 terminology for AllergyStatus to the FHIR STU3 terminology associated with AllergyIntolerance.clinicalStatus and .verificationStatus."/>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/conceptmap-AllergieStatusCodelijst-to-allergy-clinical-status.xml]: Dit moet dan tussen backticks (dus "... with `AllergyIntolerance.clinicalStatus` and ...")

File: Profiles - ZIB 2017/conceptmap-ContactTypeCodelijst-to-ActEncounterCode.xml [hunk 7]
```diff
@@ -17,8 +17,8 @@
             <use value="work"/>
         </telecom>
     </contact>
-    <description value="Maps the HCIM Contact release 2017 terminology for ContactType to the FHIR STU3 terminology associated with Encounter.class"/>
-    <purpose value="To aid in mapping from HCIM Contact value set ContactTypeCodelijst to FHIR STU3 Contact.class"/>
+    <description value="Maps the HCIM Contact release 2017 terminology for ContactType to the FHIR STU3 terminology associated with Encounter.class."/>
+    <purpose value="To aid in mapping from HCIM Contact value set ContactTypeCodelijst to FHIR STU3 Encounter.class."/>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/conceptmap-ContactTypeCodelijst-to-ActEncounterCode.xml]: Dit moet ook tussen backticks

File: Profiles - ZIB 2017/conceptmap-InterpretatieVlaggenCodelijst-To-Observation-Interpretation.xml [hunk 8]
```diff
@@ -17,8 +17,8 @@
             <use value="work"/>
         </telecom>
     </contact>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/conceptmap-InterpretatieVlaggenCodelijst-To-Observation-Interpretation.xml:19]: backticks

File: Profiles - ZIB 2017/conceptmap-MateVanKritiekZijnCodelijst-to-allergy-intolerance-criticality.xml [hunk 9]
```diff
@@ -17,8 +17,8 @@
             <use value="work"/>
         </telecom>
     </contact>
-    <description value="Maps the HCIM AllergyIntolerance release 2017 terminology for AllergyCriticalExtent to the FHIR STU3 terminology. Known issue: this ConceptMap contains concepts that should not have been included but cannot as of yet be removed due to the backwards compatibility of this ConceptMap. See https://bits.nictiz.nl/browse/MM-1731 for more details."/>
-    <purpose value="To aid in mapping from ZIB AllergyIntolerance value set MateVanKritiekZijnCodelijst to FHIR STU3 AllergyIntolerance criticality"/>
+    <description value="Maps the HCIM AllergyIntolerance release 2017 terminology for AllergyCriticalExtent to the FHIR STU3 terminology associated with AllergyIntolerance.criticality. Known issue: this ConceptMap contains concepts that should not have been included but cannot as of yet be removed due to the backwards compatibility of this ConceptMap. See https://bits.nictiz.nl/browse/MM-1731 for more details."/>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/conceptmap-MateVanKritiekZijnCodelijst-to-allergy-intolerance-criticality.xml]: backticks

File: Profiles - ZIB 2017/conceptmap-NHGTable14-to-ActEncounterCode.xml [hunk 10]
```diff
@@ -17,8 +17,8 @@
             <use value="work"/>
         </telecom>
     </contact>
-    <description value="Maps the NHG Table 14 version 7 (September 2011) terminology for ContactType to the FHIR STU3 terminology associated with Encounter.class"/>
-    <purpose value="To aid in mapping from ZIB Contact value set ContactTypeCodelijst to FHIR STU3 Contact.class"/>
+    <description value="Maps the NHG Table 14 version 7 (September 2011) terminology for ContactType to the FHIR STU3 terminology associated with Encounter.class."/>
+    <purpose value="To aid in mapping from NHG Table 14 to FHIR STU3 Encounter.class."/>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/conceptmap-NHGTable14-to-ActEncounterCode.xml]: backticks

File: Profiles - ZIB 2017/conceptmap-OrderStatusCodelijst-to-RequestStatus.xml [hunk 11]
```diff
@@ -17,8 +17,8 @@
       <use value="work"/>
     </telecom>
   </contact>
-  <description value="Maps the HCIM ProcedureRequest release 2017 terminology for OrderStatus to the FHIR STU3 terminology associated with ProcedureRequest.status"/>
-  <purpose value="To aid in mapping from HCIM ProcedureRequest value set OrderStatus to FHIR STU3 RequestStatus value set"/>
+  <description value="Maps the HCIM ProcedureRequest release 2017 terminology for OrderStatus to the FHIR STU3 terminology associated with ProcedureRequest.status."/>
+  <purpose value="To aid in mapping from HCIM ProcedureRequest value set OrderStatusCodelijst to FHIR STU3 ProcedureRequest.status."/>
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/conceptmap-OrderStatusCodelijst-to-RequestStatus.xml]: backticks

File: Profiles - ZIB 2017/zib-HelpFromOthers.xml [hunk 12]
```diff
@@ -45,7 +45,7 @@
     <element id="CarePlan.intent">
       <path value="CarePlan.intent" />
       <short value="plan" />
-      <comment value="Although the Zib is not really a (care)plan, this resources falls almost completly inline with the Zib. From the 4 possible codes here plan comes closest. Therefore this element is fixed to plan." />
+      <comment value="Although the HCIM is not really a (care) plan, this resource falls almost completely inline with the HCIM. From the 4 possible codes here, 'plan' comes closest. Therefore this element is fixed to 'plan'." />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-HelpFromOthers.xml]: Should be _plan_, not 'plan'.

**Review by pieter-edelman-nictiz — CHANGES_REQUESTED** [review]

**Review by pieter-edelman-nictiz — APPROVED** [review]
Ok, nou heb ik echt niks meer te klagen. Goed bezig!

**Pull request merged**